Dockerfile

```
@@ -26,9 +26,9 @@ RUN yarn install
 #
 # ---- Test ----
 # run linters, setup and tests
-# FROM dependencies AS test
-# COPY . .
-# RUN  yarn lint && yarn build
+FROM dependencies AS test
+COPY . .
+RUN yarn lint && yarn build && yarn jest
```

Let's move the order of commands to lint + test + build and I think there is a typo, `jest` instead of `test`
```suggestion
RUN yarn lint && yarn test && yarn build
```